src/dmd/libomf.d

```diff
@@ -167,29 +167,28 @@ final class LibOMF : Library
 
     /*****************************************************************************/
 
-    void addSymbol(OmfObjModule* om, const(char)* name, int pickAny = 0)
+    void addSymbol(OmfObjModule* om, const(char)[] name, int pickAny = 0)
     {
         static if (LOG)
         {
-            printf("LibOMF::addSymbol(%s, %s, %d)\n", om.name.ptr, name, pickAny);
+            printf("LibOMF::addSymbol(%s, %s, %d)\n", om.name.ptr, name.ptr, pickAny);
```

Comment:
When changing ptr to arrays, please update the `printf` to use `%.*s` instead of just `%s`.
The reason is that there is a dependency on null-termination with `%s`. This is always obvious with a pointer, but when switching to arrays, this information gets lost, and no one wants to be the poor soul debugging the bugs happening due to a missing `\0` in a `char[]`.